support/k8s/k8s-workload-registrar/README.md

```diff
@@ -65,12 +66,13 @@ and delete registration entries for workloads running on those pods. The
 workload registration entries are configured to run on any node in the
 cluster.
 
-There are two workload registration modes.
+There are three workload registration modes.
+If you use Service Account Based, don't specify both `pod_label` and `pod_annotation`. If you use Label Based, specify only `pod_label`. If you use Annotation Based, specify only `pod_annotation`.
```

Comment:
nit: `don't specify either ... or ...`

Reply:
I fixed it with https://github.com/spiffe/spire/pull/1047/commits/79f65711f6b93943e6ca16b8b02c1e4c4cd39b6f .